README.md

```diff
@@ -1 +1,58 @@
-# [WIP] Vividus extension for Visual Studio Code
\ No newline at end of file
+# Development guide
+
+## Tools
+* Node JS v12.13.0 or higher
+* Open JDK 12.0.1 or higher
+* Eclipse IDE 2019-09 R (4.13.0) or higher
+* Visual Studio Code 1.47.1 or higher
+
+## General
+1. Run `git clone https://github.com/vividus-framework/vividus-studio.git`
+2. Run `npm install -g yo generator-code`
+5. Install `Eclipse PDE (Plug-in Development Environment)` plugin in Eclipse IDE through `Eclipse Marketplace`
```

Comment:
I could not get the plugin working on on Ubuntu 20, and I had to install Eclipse for plugin developers